components/ComInterface/src/de/ees/group1/bt/BT_device.java

```diff
@@ -81,22 +83,30 @@ public boolean startService(){
 		
 	}
 	
-	public boolean sendMessage(byte[] message){
+	public boolean sendMessage(Telegramm<? extends Serializable> message){
 	
+		ObjectOutput out = null;
+		
 		try {
-			
-			//TODO
-			//TODO Längenbytes hnzufügen
-			//TODO
-			this.dos.write(message);
+			out = new ObjectOutputStream(this.dos);
```

Comment:
Muss der Output Stream in dieser Methode erstellt werden? Es wird so bei jedem Methodenaufruf ein neuer Output-Stream instanziiert. Besser wäre, wenn ein und der selbe Output-Stream mehrfach genutzt wird.